qt/scientific_interfaces/Indirect/JumpFitModel.cpp

```diff
@@ -364,6 +364,8 @@ std::string JumpFitModel::singleFitOutputName(std::size_t index,
 
 std::string JumpFitModel::getResultXAxisUnit() const { return ""; }
 
+std::string JumpFitModel::getResultLogValue() const { return "SourceName"; }
```

Comment:
`SourceName` was previously referred to as LogName, I am confused as to why it is now returned as the LogValue

Reply:
Good question, I think the surrounding code in QENSFitSequential uses `LogValue` instead of `LogName` in order to stay consistent with the `LogValue` property seen in the `PlotPeakByLogValue` algorithm. It might make more sense to change all of these to `LogName`, do you think it might be worth doing this?

Reply:
@DTasev It seems `PlotPeakByLogValue` and its `LogValue` property is used within the `FitPropertyBrowser`. I think I should avoid changing this, instead I will just change the QENSFitSequential algorithm and the IndirectFittingModel mentions of `LogValue` to `LogName`

Reply:
I see, I think this change makes it a bit easier to understand